packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -80,10 +93,26 @@ const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = {
     },
     carousel_sequenceIndicator: {
         display: "inline-block",
-        padding: "0 2px",
+        padding: "4px",
+        position: "relative",
         "&:focus": {
             outline: "none",
         },
+        ...applyFocusVisible<DesignSystem>({
+            "&::after": {
+                opacity: "1",
+                position: "absolute",
+                border: "2px solid transparent",
```

Comment:
should this map to the design system? Is the border for the indicators mapped? If so, I think this should follow as well.

Reply:
I wasn't sure entirely. @bheston The sequence indicators have a hard coded width and the height maps to `designUnit` since these elements are so close to them bottom what happens when density increases? Seems that if the height and focus border increase they will clip outside the bounds of the carousel or get very close.

Just bumped the design to 3 and carousel looks very wrong:
![image](https://user-images.githubusercontent.com/37851214/75195960-d2253080-570f-11ea-9173-2236f543bd8e.png)